### PR TITLE
Add common docker and kubernetes packages

### PR DESCRIPTION
**Pull request by kvch**

### CODE REVIEW

**Comment by kvch** [comment]
jenkins run tests

File: kubernetes/informer.go [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+}
+
+func nameSelector(options *metav1.ListOptions, name string) {
+	if name != "" {
+		options.FieldSelector = "metadata.name=" + name
+	}
+}
+
+// NewInformer creates an informer for a given resource
+func NewInformer(client kubernetes.Interface, resource Resource, opts WatchOptions, indexers cache.Indexers) (cache.SharedInformer, string, error) {
```
Comment (cmacknz) [inline kubernetes/informer.go:44]: Can we include these changes? https://github.com/elastic/beats/pull/30699/files

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
-# elastic-agent-autodiscover
\ No newline at end of file
+# elastic-agent-autodiscover
+
+This repo contains packages required by autodiscover.
+
+* `github.com/elastic/elastic-agent-autodiscover/bus`
+* `github.com/elastic/elastic-agent-autodiscover/docker`
+* `github.com/elastic/elastic-agent-autodiscover/kubernes`
```
Comment (cmacknz) [inline README.md:7]:
```suggestion
* `github.com/elastic/elastic-agent-autodiscover/kubernetes`
```

**Pull request merged**